doc/src/compute_efield_wolf_atom.rst

```diff
@@ -0,0 +1,127 @@
+.. index:: compute efield/wolf/atom
+
+compute efield/wolf/atom command
+================================
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   compute ID group-ID efield/wolf/atom alpha keyword val
+
+* ID, group-ID are documented in :doc:`compute <compute>` command
+* efield/atom/wolf = style name of this compute command
+* alpha = damping parameter (inverse distance units)
+* zero or more keyword/value pairs may be appended
+* keyword = *limit* or *cutoff*
+
+  .. parsed-literal::
+
+     *limit* group2-ID = limit computing the electric field contributions to a group (default: all)
+     *cutoff* value = set cutoff for computing contributions to this value (default: maximum cutoff of pair style)
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   compute 1 all efield/wolf/atom 0.2
+   compute 1 mols efield/wolf/atom 0.25 limit water cutoff 10.0
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+Define a computation that approximates the electric field at each atom in a group.
+
+.. math::
+
+   \vec{E}_i = \frac{\vec{F}coul_i}{q_i} = \sum_{j \neq i} \frac{q_j}{r_{ij}^2} \qquad r < r_c
+
+The electric field at the position of the atom *i* is the coulomb force
+on a unit charge at that point, which is equivalent to dividing the
+Coulomb force by the charge of the individual atom.
+
+In this compute the electric field is approximated as the derivative of
+the potential energy using the Wolf summation method, described in
+:ref:`Wolf <Wolf4>`, given by:
+
+.. math::
+   E_i = \frac{1}{2} \sum_{j \neq i}
+   \frac{q_i q_j {\rm erfc}(\alpha r_{ij})}{r_{ij}} +
+   \frac{1}{2} \sum_{j \neq i}
+   \frac{q_i q_j {\rm erf}(\alpha r_{ij})}{r_{ij}} \qquad r < r_c
+
+where :math:`\alpha` is the damping parameter, and *erf()* and *erfc()*
+are error-function and complementary error-function terms.  This
+potential is essentially a short-range, spherically-truncated,
+charge-neutralized, force-shifted, pairwise *1/r* summation.  With a
+manipulation of adding and subtracting a self term (for i = j) to the
+first and second term on the right-hand-side, respectively, and a small
+enough :math:`\alpha` damping parameter, the second term shrinks and the
+potential becomes a rapidly-converging real-space summation.  With a
+long enough cutoff and small enough :math:`\alpha` parameter, the
+electric field calculated by the Wolf summation method approaches that
+computed using the Ewald sum.
+
+The value of the electric field components will be 0.0 for atoms not in
+the specified compute group.
+
+When the *limit* keyword is used, only contributions from atoms in the
```

Comment:
The keyword name *limit* is not very descriptive for this option. Is there a similar option in some existing fix or compute that could be copied.  If not, what about *exclude* or *exclude/other*

Reply:
@athomps I just did a bit of a survey and in most cases either the second group argument is mandatory or optional but without a keyword. Compute coord/atom uses the keyword "group" which is not really representative of what it does, either. Perhaps "restrict" or "select" could be an option, but if we want to go for consistency, then "group" would be the precedence. The terms "limit" or "restrict" or "select" are indicating that the second group is limiting which atoms are contributing to the field while the compute group is selecting for which atoms the property is computed.